lib/utils/dateTestUtils.ts

```
@@ -0,0 +1,326 @@
+  formatDateForInput,
+  getDateValidationInfo,
+  isValidDateString,
+  isFutureDate,
+  isTooOldDate,
+} from "./dateUtils";
+
+export interface DateTestResult {
+  input: string | Date | null | undefined;
+  formatted: string | null;
+  isValid: boolean;
+  validation: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)